docs/source/index.rst

```diff
@@ -275,6 +277,8 @@ TensorFlow and/or Flax.
 +-----------------------------+----------------+----------------+-----------------+--------------------+--------------+
 |     Funnel Transformer      |       ✅       |       ✅       |       ✅        |         ✅         |      ❌      |
 +-----------------------------+----------------+----------------+-----------------+--------------------+--------------+
+|           GPTNeo            |       ✅       |       ✅       |       ✅        |         ❌         |      ❌      |
```

Comment:
Why is the model listed as not supporting TF? It's written in TF. If this is on the HF side, is there anything we can give you to make a TF implementation easier?